scripts/ci/tags.yaml

```diff
@@ -40,6 +40,28 @@
 #    to get test coverage as broad as possible.
 # 2. Keep tag entries sorted alphabetically
 
+bluetooth:
+    files:
+        - drivers/bluetooth/
+        - subsys/bluetooth/
+        - subsys/net/l2/bluetooth/
+
+cmsis_dsp:
+    files:
+        - modules/Kconfig.cmsis_dsp
+# we have no means of telling file changes in a module, so for assume any
+# change to west.yml is updating something we need to re-test for.
+        - west.yml
+
 kernel:
     files:
         - kernel/
+
+posix:
+    files:
+        - lib/posix
+
+prf:
```

Comment:
please use this opportunity to fix that tag, 'prf' is an old tag, I would change this to cbprintf now, which is being set in other tests.

Reply:
Done